lib/smoke_detector/providers/rollbar.rb

```diff
@@ -42,14 +42,13 @@ def default_client_settings
         </script>
       JS
```

Comment:
It appears to be added to the response body via middleware

https://github.com/lumoslabs/smoke_detector/blob/master/lib/smoke_detector/middleware/javascript_monitors.rb